test/manipulation.test.js

```diff
@@ -743,7 +743,15 @@ describe('manipulation', function() {
       Todo = db.define('Todo', {
         content: String,
       });
-      db.automigrate(['Post', 'Todo'], done);
+      Person = db.define('Person', {
+        name: String,
+        gender: String,
+        married: Boolean,
+        age: {type: Number, index: true},
+        dob: Date,
+        createdAt: {type: Date, default: Date},
```

Comment:
Are all of these properties required by the newly added tests? Please keep the size of test models as small as feasible, define only properties that are used by the tests.